### PR TITLE
[ohm-plugins-misc] Updated Telepathy Call channel interface path.

### PR DESCRIPTION
**Pull request by tswindell**
Signed-off-by: Tom Swindell <tom.swindell@jollamobile.com>

### CODE REVIEW

File: plugins/telephony/ohm/telephony.h [hunk 1]
```diff
@@ -37,7 +37,7 @@ USA.
 #define TP_CHANNEL_DTMF   TP_CHANNEL".Interface.DTMF"
 #define TP_CHANNEL_MEDIA  TP_CHANNEL".Type.StreamedMedia"
 
-#define TP_CHANNEL_CALL_DRAFT TP_CHANNEL".Type.Call.DRAFT"
```
Comment (rburchell) [inline plugins/telephony/ohm/telephony.h]: it's no longer a draft, so at the least this should probably be renamed

Comment (tswindell) [inline plugins/telephony/ohm/telephony.h]: Ah, good point. Will revise and resubmit.

Comment (tswindell) [inline plugins/telephony/ohm/telephony.h]: Done

**Comment by rburchell** [comment]
looks functionally ok. I would like to see details of why you're doing what you're doing in the commit message (a good commit message contains the what in the summary - which you have - and then a description about the why/how it later on for anyone who needs to follow the trail later on)

i'm also curious, are there no actual changes apart from the path changing?

**Comment by tswindell** [comment]
After managing to find an old cached copy of the Call.Draft specification, not major differences jump out at me. Functionally it seems to be working as expected at the moment, but there may indeed be some more changes incoming if I find something that is indeed broken.

**Comment by jusa** [comment]
@tswindell can you add that bit of info to the commit message as well? I'll then merge the patch.

**Comment by jusa** [comment]
Thanks, merging.

**Pull request merged**